commands/monitoring.go

```diff
@@ -0,0 +1,350 @@
+	AddStringFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicyDescription, "", "", "A description of the alert policy.")
+	AddStringFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicyType, "", "", "The type of alert policy.")
+	AddStringFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicyCompare, "", "", "The comparator of the alert policy. Either `GreaterThan` or `LessThan`")
+	AddStringFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicyWindow, "", "5m", "The window to apply the alert policy conditions against.")
+	AddIntFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicyValue, "", 0, "The value of the alert policy to compare against.")
+	AddBoolFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicyEnabled, "", true, "Whether the alert policy is enabled.")
+	AddStringSliceFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicyEmails, "", nil, "Emails to send alerts to.")
+	AddStringSliceFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicyTags, "", nil, "Tags to apply the alert against.")
+	AddStringSliceFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicyEntities, "", nil, "Entities to apply the alert against.")
+	AddStringSliceFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicySlackChannels, "", nil, "Slack channels to send alerts to.")
+	AddStringSliceFlag(cmdAlertPolicyUpdate, doctl.ArgAlertPolicySlackURLs, "", nil, "Slack URLs to send alerts to.")
+
+	CmdBuilder(cmd, RunCmdAlertPolicyGet, "get <alert-policy-uuid>", "Retrieve information about an alert policy", `Use this command to retrieve an alert policy and see its configuration.`, Writer,
+		displayerType(&displayers.AlertPolicy{}))
+
+	CmdBuilder(cmd, RunCmdAlertPolicyList, "list", "List all alert policies", `Use this command to retrieve a list of all the alert policies in your account.`, Writer,
+		aliasOpt("ls"), displayerType(&displayers.AlertPolicy{}))
+
+	cmdRunAlertPolicyDelete := CmdBuilder(cmd, RunCmdAlertPolicyDelete, "delete <alert-policy-uuid>...", "Delete an alert policy", `Use this command to delete an alert policy.`, Writer)
+	AddBoolFlag(cmdRunAlertPolicyDelete, doctl.ArgForce, doctl.ArgShortForce, false, "Delete an alert policy without confirmation prompt")
+
+	return cmd
+}
+
+// RunCmdAlertPolicyCreate runs alert policy create.
+func RunCmdAlertPolicyCreate(c *CmdConfig) error {
+	ms := c.Monitoring()
+
+	desc, err := c.Doit.GetString(c.NS, doctl.ArgAlertPolicyDescription)
+	if err != nil {
+		return err
+	}
+	alertType, err := c.Doit.GetString(c.NS, doctl.ArgAlertPolicyType)
+	if err != nil {
+		return err
+	}
+	value, err := c.Doit.GetInt(c.NS, doctl.ArgAlertPolicyValue)
+	if err != nil {
+		return err
+	}
+
+	window, err := c.Doit.GetString(c.NS, doctl.ArgAlertPolicyWindow)
+	if err != nil {
+		return err
+	}
+
+	entities, err := c.Doit.GetStringSlice(c.NS, doctl.ArgAlertPolicyEntities)
+	if err != nil {
+		return err
+	}
+
+	tags, err := c.Doit.GetStringSlice(c.NS, doctl.ArgAlertPolicyTags)
+	if err != nil {
+		return err
+	}
+
+	enabled, err := c.Doit.GetBoolPtr(c.NS, doctl.ArgAlertPolicyEnabled)
```

Comment:
```suggestion
	enabled, err := c.Doit.GetBool(c.NS, doctl.ArgAlertPolicyEnabled)
```

Looks like `GetBoolPtr` ignores defaults and and excluding the enabled flag results in `nil`. Not sure if that's actually the intended behavior. 

https://github.com/digitalocean/doctl/blob/c7bf8d427e73e395f59f743cee05f6747392b677/doit.go#L292-L298

Reply:
good call out!